package-lock.json

```diff
@@ -1,2939 +0,0 @@
-{
```

Comment:
What is the motivation for deleting this file?

Reply:
A package-lock file implies npm is used, instead of yarn. Since this repo uses yarn, it should only have a yarn.lock file. There's a warning to that effect when you call `yarn start`